streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -146,6 +146,7 @@
 
     /** {@code client.id} */
     public static final String CLIENT_ID_CONFIG = CommonClientConfigs.CLIENT_ID_CONFIG;
+    private static final String CLIENT_ID_DOC = "An ID prefix string used for the client IDs of internal consumer, producer and restore-consumer, with pattern '<client.id>-StreamThread-< threadSequenceNumber >-<consumer|producer|restore-consumer>'.";
```

Comment:
Maybe one more nit (\cc @dguy ): remove the ` ` before and after `threadSequenceNumber` (I added those blanks in my Github comment only to get the right rendering. Otherwise, Github did some weird rendering of my comment. It's not supposed to be part if the message.

Also, can we break this line into multiple... we should try to limit line length to 120 chars. Thx.